src/components/TablePagination/util.ts

```diff
@@ -0,0 +1,40 @@
+// Logic for showing visible page buttons based on input of current page and total page
+// Based on USWDS guidelines - https://designsystem.digital.gov/components/pagination/
+
+// Creates an array of page numbers starting a 1, from an input of a number
+const fillMinPages = (totalPages: number) => {
+  return Array.from({ length: totalPages }, (_, i) => i + 1);
+};
+
+const getVisiblePages = (currentPage: number, totalPages: number) => {
+  // If total pages are less than 7 (minimum for elipsis), create an array of available page based on total
+  if (totalPages < 7) {
+    return fillMinPages(totalPages);
+  }
+  // If total is more than the minimum for elipsis (7) and IS adjacent to the end page within 4 pages
+  // Returns an array with the start page, and the last four pages
+  if (
+    currentPage % 4 >= 0 &&
```

Comment:
Not sure what the `currentPage % 4 >= 0` checks are for; shouldn't that always be true if `currentPage > 0`?

Reply:
Updated